Failing `test_proxy.py::test_sizeof_cudf`

I'm seeing `dask_cuda/tests/test_proxy.py::test_sizeof_cudf` failing locally:

```python
___________________________________________________ test_sizeof_cudf ____________________________________________________

    def test_sizeof_cudf():
        cudf = pytest.importorskip("cudf")
        a = cudf.datasets.timeseries().reset_index()
        a_size = sizeof(a)
        pxy = proxy_object.asproxy(a)
        assert a_size == pytest.approx(sizeof(pxy))
        pxy._pxy_serialize(serializers=("dask",))
        assert a_size == pytest.approx(sizeof(pxy))
        assert pxy._pxy_get().is_serialized()
        pxy._pxy_cache = {}
>       assert a_size == pytest.approx(sizeof(pxy))
E       assert 86832408 == 87156456 ± 8.7e+01
E         +86832408
E         -87156456 ± 8.7e+01

dask_cuda/tests/test_proxy.py:609: AssertionError
================================================ short test summary info ================================================
FAILED dask_cuda/tests/test_proxy.py::test_sizeof_cudf - assert 86832408 == 87156456 ± 8.7e+01
=========================================== 1 failed, 91 deselected in 5.19s ============================================
```

The issue above is present with the following cuDF build:

```
cudf                      22.02.00a220110 cuda_11_py38_g0d5ec7f890_245    rapidsai-nightly
dask-cudf                 22.02.00a220110 cuda_11_py38_g0d5ec7f890_245    rapidsai-nightly
libcudf                   22.02.00a220110 cuda11_g0d5ec7f890_245    rapidsai-nightly
```

But an old environment with the following passes:

```
cudf                      22.02.00a220106 cuda_11_py38_g33f7f0d032_227    rapidsai-nightly
dask-cudf                 22.02.00a220106 cuda_11_py38_g33f7f0d032_227    rapidsai-nightly
libcudf                   22.02.00a220106 cuda11_g33f7f0d032_227    rapidsai-nightly
```

@shwina @quasiben did we have any changes in cuDF that could have affected the way `sizeof` in any of its objects?

cc @madsbk

--- a/dask_cuda/tests/test_proxy.py
+++ b/dask_cuda/tests/test_proxy.py
@@ -605,6 +605,8 @@ def test_sizeof_cudf():
     pxy._pxy_serialize(serializers=("dask",))
     assert a_size == pytest.approx(sizeof(pxy))
     assert pxy._pxy_get().is_serialized()
+    # By clearing the cache, `sizeof(pxy)` now measure the serialized data
+    # thus we have to increase the tolerance.
     pxy._pxy_cache = {}
-    assert a_size == pytest.approx(sizeof(pxy))
+    assert a_size == pytest.approx(sizeof(pxy), rel=1e-2)
     assert pxy._pxy_get().is_serialized()

